ui/admin/tests/acceptance/accounts/selectors.js

```diff
@@ -6,8 +6,25 @@
 export const APP_HEADER_DROPDOWN =
   '.rose-header-utilities details:first-child summary';
 
+export const MANAGE_DROPDOWN =
+  '[data-test-manage-auth-methods-dropdown] button';
+export const MANAGE_DROPDOWN_NEW_ACCOUNT = (url) =>
+  `[data-test-manage-auth-methods-dropdown] a[href="${url}"]`;
+
 export const FIELD_CURRENT_PASSWORD = '[name=currentPassword]';
 export const FIELD_CURRENT_PASSWORD_VALUE = 'current password';
 
 export const FIELD_NEW_PASSWORD = '[name=newPassword]';
 export const FIELD_NEW_PASSWORD_VALUE = 'new password';
+
+export const FIELD_NAME = '[name=name]';
```

Comment:
this selector also exists in the common selectors file. Is there a reason to have it in both files ?

Reply:
Nope. I just accidentally duplicated it. I will update it now. Thanks for catching.

Reply:
thank you
im also noticing that `FIELD_DESCRIPTION` (perhaps `FIELD_DESCRIPTION_VALUE` as well) is defined in this file but perhaps it should be moved to common selectors

Reply:
Yep I noticed that as well and moved it over. Name and description for sure are common since we use it for every resource